content/en/docs/reference/glossary/service-account.md

```diff
@@ -15,5 +15,4 @@ tags:
 
 <!--more--> 
 
-When processes inside Pods access the cluster, they are authenticated by the API server as a particular service account, for example, `default`. When you create a Pod, if you do not specify a service account, it is automatically assigned the default service account in the same namespace {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
-
+When processes inside Pods access the cluster, they are authenticated by the API server as a particular ServiceAccount, for example, `default`. When you create a Pod, if you do not specify a service account, it is automatically assigned the default service account in the same namespace {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
```

Comment:
If we treat ServiceAccount as a term, we should be using it consistently, i.e. substitute all "service account" by "ServiceAccount".
If "service account" is treated as a term and we spell ServiceAccount as the resource name when appropriate, we should keep the title as is.

Reply:
> If we treat ServiceAccount as a term, we should be using it consistently, i.e. substitute all "service account" by "ServiceAccount".

Why?

English is a special localization. The English term for the ServiceAccount API object is “service account”. In other locales it's fine to mix the object name and the natural language name. To me it's also OK to do that in English.

Reply:
@sftim I raised the concern because we have both "ServiceAccount" and "service account" in the same paragraph. If "service account" is a term, we may want to change the title back to "service account". After all, a glossary entry is about a term, not the resource name. Just my $0.02.

Reply:
I [suggested](https://github.com/kubernetes/website/pull/17997) that the Korean localization team write “ServiceAccount”, with no space, in the Korean localization of this term, and they declined to accept that change whilst upstream has the definition as two English words.

Sounds like there's a challenge here. My aim here is for people reading localized documentation to easily be able to see the name of the resource being described.

Reply:
Essentially, I see
> When you create a Pod, if you do not specify a service account, it is automatically assigned the default service account in the same namespace {{< glossary_tooltip text="Namespace" term_id="namespace" >}}

as text that could localize to any target language and that would not use the resource name ServiceAccount when localized.

Reply:
Maybe we have different understandings of the "glossary". It is a glossary for terms or is it a glossary for resource names. If we are defining glossary for terms, we should:

- use "service account" everywhere except for the special case where we are referring to the API resource which is named `ServiceAccount` by happy accident;
- use "service account" as the title for this page, not `ServiceAccount`.

If otherwise we are providing an inventory of API resource names, we should:

- use `ServiceAccount` as the title for this page;
- focus on the `ServiceAccount`, and use "service account" as a generic phrase.

I'm more leaning towards the former.

Reply:
It sounds like the page should also mention that the API resource for a “service account” is ServiceAccount. Does that seem right?

Reply:
Sounds right to me. :)

Reply:
Revised; how does this look?